frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -240,21 +240,50 @@ export const getReportBugLink = (cv: ClusterVersionKind): { label: string; href:
     return null;
   }
 
-  // Show a Bugzilla link for prerelease versions and a support case link for supported versions.
   const { major, minor, prerelease } = parsed;
-  const bugzillaVersion = major === 4 && minor <= 3 ? `${major}.${minor}.0` : `${major}.${minor}`;
-  const environment = encodeURIComponent(`Version: ${version}
+  let productName;
+  switch (window.SERVER_FLAGS.branding) {
+    case 'openshift':
+    case 'ocp':
+      productName = 'OpenShift%20Container%20Platform';
```

Comment:
I would leave these unencoded

```suggestion
      productName = 'OpenShift Container Platform';
```

then call `encodeURIComponent` when adding them to the URL below.

Reply:
Done.